internal/cmd/beta/server/backup/volume-backup/delete/delete.go

```diff
@@ -0,0 +1,118 @@
+package delete
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/serverbackup/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/serverbackup"
+)
+
+const (
+	volumeBackupIdArg = "VOLUME_BACKUP_ID"
+	serverIdFlag      = "server-id"
+	backupIdFlag      = "backup-id"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	BackupId string
+	VolumeId string
+	ServerId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("delete %s", volumeBackupIdArg),
+		Short: "Deletes a Server Volume Backup. Operation always is async.",
```

Comment:
```suggestion
		Short: "Deletes a Server Volume Backup",
```

Reply:
I think this doesn't need to be in the short description, which is shown in the help of the parent command

Reply:
Could you also remove it from other commands?